spec/models/user_spec.rb

```diff
@@ -29,4 +29,13 @@
       expect(FactoryGirl.create(:user)).to be_valid
     end
   end
+
+  describe 'Restaurant Owner role' do
+    params = {email: 'xavy@ibm.com', password: '12345678', password_confirmation: '12345678', restaurant_owner: true  }
+    user = FactoryGirl.create(:user, params)
+    let (:current_user) {user}
```

Comment:
I would trust the factory to do its job and only set the `restaurant_owner` attribute:

```ruby
let (:current_user) { FactoryGirl.create(:user, restaurant_owner: true }
```
That vill save you a few lines of code and cleaner syntax.